packages/browser-sdk/src/feature/features.ts

```diff
@@ -92,6 +92,8 @@ export function flattenJSON(obj: Record<string, any>): Record<string, any> {
       for (const flatKey in flat) {
         result[`${key}.${flatKey}`] = flat[flatKey];
       }
+    } else if (typeof obj[key] === "undefined") {
```

Comment:
JSON has no concept of `undefined` so it would not be serialized anyway, but as you said - let's play nice :)

Reply:
We don’t actually serialize to JSON in this case. The context of flattened and then sent as search parameters.